cpp/src/arrow/acero/aggregate_node.cc

```diff
@@ -77,6 +78,19 @@ namespace acero {
 
 namespace {
 
+template <typename KernelType>
+struct ARROW_ACERO_EXPORT AggregateNodeArgs {
```

Comment:
I am actually not sure what does `ARROW_ACERO_EXPORT` does but it seems this is a private struct of the cc file and doesn't need to be exported?

Reply:
Yes, this was moved here from the header but has no effect here - should be removed.

Reply:
Fixed.